ext/data/templates/default-anki-field-templates.handlebars

```diff
@@ -302,6 +302,54 @@
     {{~/if~}}
 {{/inline}}
 
+{{#*inline "frequency-number"}}
+    {{~#scope~}}
+        {{~set "result-freq" -1 ~}}
+        {{~set "prev-freq-dict" "" ~}}
+        {{~set "t" 1 ~}}
+
+        {{~#each definition.frequencies~}}
+            {{~set "read-freq" false ~}}
+            {{~#if (op "!==" (get "prev-freq-dict") this.dictionary ) ~}}
+                {{~set "read-freq" true ~}}
+                {{~set "prev-freq-dict" this.dictionary ~}}
+            {{/if~}}
+
+            {{~#if (get "read-freq") ~}}
+                {{~#set "numericFrequencyMatch"}}{{~#regexMatch "\d+" ""}}{{~this.frequency~}}{{/regexMatch~}}{{/set~}}
```

Comment:
Consistent naming scheme for variables. Looks like the precedent is `camelCase` for `get/set` in the current templates, so should probably go with that.